Documentation Changes: Edits to Introduction and improvements to layout.

awholegunch opened the pull request:
Fixes problem with hiding relations in footer.
Fixes a problem in the Introduction (and elsewhere) in showing output samples that conflicted with the reST syntax for replacements (`|-`, `|+`).
Solves a problem with the tabbed-parsed-includes that prevented showing CLI output in cases where a table was being shown.
Corrected a grammatical mistake in the introduction.
Corrected an example of `curl` that was in a different style of quoting than our examples.

As we are now an Open Source project whose documentation depends heavily on the contributions of the open-source Sphinx team, per their request, adds `Created using Sphinx 1.3.1.` to the footer of each page, beside our copyright. Please comment if there are any issues with that.

Passes [Quick Build 4](http://builds.cask.co/browse/CDAP-DQB21-4)

[Revised Introduction](http://builds.cask.co/artifact/CDAP-DQB21/shared/build-4/Docs-HTML/3.5.0-SNAPSHOT/en/introduction/index.html)

Commit:
Edits to Introduction and improvements to layout. Fixes problem with hiding relations in footer.

awholegunch commented:
"On-hold" as I found an error.

Commit:
Correct replacements.

Commit:
Use quotes identical with those shown in examples.

Commit:
Merge branch 'develop' into docs/3.5_Fix_Intro

# Conflicts:
#	cdap-docs/_common/tabbed-parsed-literal.py
# Resolved in merge

anwar6953 commented:
Code changes LGTM.

Is this a problem we want to try to tackle? If a table is too wide, instead of making just the table scrollable, the entire page becomes wider to fit it.

![image](https://cloud.githubusercontent.com/assets/2440977/16205366/4f16de06-36d8-11e6-9b5c-25a54cf0f413.png)

awholegunch commented:
This issue comes about partially because the CDAP CLI doesn't accept a continuation character in the entry of commands. I'm sure there are reasons for that (the particular text entry code used, etc.) but that's why I haven't broken the line, as I would with a `curl` command example. And these examples are intended to be copy and pasted directly for ease of use.

That said, I will look into making the page larger for this page.

Commit:
Clean-up the FAQs style code. Add an "Intro" theme with a background color of white
to solve problem of tables that go large.

awholegunch commented:
I have added a new theme ("cdap-intro") for the intro page that has a white background. The table sizes are still the same, as we would want the first paragraph of text to not become overly wide. (It would be impossible to read then.) I also cleaned-up the FAQ section code as it had some duplicated material that can be removed.

See for an update: http://builds.cask.co/artifact/CDAP-DQB21/shared/build-4/Docs-HTML/3.5.0-SNAPSHOT/en/introduction/index.html

anwar6953 commented:
@awholegunch I meant: is it possible to have just the really wide table/command be scrollable, instead of making the whole page wider?

Edit: I guess the problem is gone, now that the background is white.

anwar6953 commented on cdap-docs/_common/_themes/cdap-intro/theme.conf:
```diff
@@ -0,0 +1,17 @@
+manual_icons = []
+meta_git = []
+stickysidebar = true
+release = ''
```
we need these empty strings?

awholegunch replied:
Yes, they set the options, and show (document) what type they should be. They are then overwritten in particular uses of the theme. And, if they aren't, they are an empty string, rather than a `None`, which would cause problems when they are added to a string...

anwar6953 commented on cdap-docs/_common/_themes/cdap-faqs/theme.conf:
```diff
@@ -4,10 +4,14 @@ stylesheet = cdap-faqs.css
 stickysidebar = true
-versions = []
-versions_data = ''
+release = ''
```
we need these empty strings?

awholegunch replied:
see above.

anwar6953 commented:
2 (minor) questions.
Remainder LGTM

anwar6953 commented:
LGTM
